src/variables.tf

```diff
@@ -305,6 +305,11 @@ variable "app_gateway_management_certificate_name" {
   description = "Application gateway api management certificate name on Key Vault"
 }
 
+variable "app_gateway_sku_tier" {
+  type        = string
+  description = "Application gateway additional support for autoscaling, zone redundancy and Static VIP"
```

Comment:
for vars description u can copy from terraform provider page :) https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/application_gateway#tier